summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/store/DirectoryBatchedStore.scala

```diff
@@ -81,7 +81,7 @@ class DirectoryBatchedStore[K <: Writable, V <: Writable](val rootPath: String)(
 
         val lastBatchStatus =
           hdfsPaths.map(getFileStatus(_, conf))
-            .filter { input => input._1 && (BatchID(input._2) < exclusiveUB) }
+            .filter { input => input._1 && (BatchID(input._2.milliSinceEpoch) < exclusiveUB) }
```

Comment:
this does not seem correct to me. You shouldn't just pass a millis to a `BatchID`. That's what Batchers do. Why is this change here?